app/pages/project/classify.cjsx

```diff
@@ -215,12 +215,12 @@ module.exports = React.createClass
   scrollIntoView: (e) ->
     # Auto-scroll to the middle of the classification interface on load.
     # It's not perfect, but it should make the location of everything more obvious.
-    lineHeight = parseFloat getComputedStyle(document.body).lineHeight
+    lineHeight = parseFloat getComputedStyle(document?.body).lineHeight
     el = @getDOMNode()
     space = (innerHeight - el.offsetHeight) / 2
     idealScrollY = el.offsetTop - space
-    if Math.abs(idealScrollY - scrollY) > lineHeight
-      animatedScrollTo document.body, el.offsetTop - space, 333
+    #if Math.abs(idealScrollY - scrollY) > lineHeight
+      #animatedScrollTo document?.body, el.offsetTop - space, 333
```

Comment:
I'm all for removing this feature, but not by commenting it out (and probably not in this PR).